### PR TITLE
Added environment variable for test env

### PR DESCRIPTION
**Pull request by NehaNaithani**

### CODE REVIEW

File: iofog-agent-daemon/src/main/java/org/eclipse/iofog/tracking/TrackingInfoUtils.java [hunk 1]
```diff
@@ -18,7 +18,7 @@ public static JsonObject getStartTrackingInfo() {
         try {
             String gpsCoordinates = Configuration.getGpsCoordinates();
             String gpsMode = Configuration.getGpsMode().name();
-            boolean developerMode = Configuration.isDeveloperMode();
+            boolean developerMode = Configuration.isSecureMode();
```
Comment (neilotoole) [inline iofog-agent-daemon/src/main/java/org/eclipse/iofog/tracking/TrackingInfoUtils.java]: Should this be the opposite? As in, `developerMode` is the opposite of `secureMode`, or am I not understanding what secure mode means?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -190,6 +190,7 @@ jobs:
 
       - script: |
           cd packaging/iofog-agent
+          sed -i 's/default/dev/g'  etc/iofog-agent/config-switcher_new.xml
```
Comment (sergerad) [inline azure-pipelines.yml]: Does this work on mac and linux? In my experience need `-i.bkp` for linux. Need to specify backup file suffix

File: iofog-agent-daemon/src/main/java/org/eclipse/iofog/tracking/TrackingInfoUtils.java (hunk 1 above)
Comment (sergerad) [inline iofog-agent-daemon/src/main/java/org/eclipse/iofog/tracking/TrackingInfoUtils.java]: boolean secureMode =

Comment (NehaNaithani) [reply iofog-agent-daemon/src/main/java/org/eclipse/iofog/tracking/TrackingInfoUtils.java]: updated

File: azure-pipelines.yml (hunk 2 above)
Comment (NehaNaithani) [reply azure-pipelines.yml]: yeah this won't work. `sed -i '' 's/default/dev/g' etc/iofog-agent/config-switcher_new.xml` should be used or `-i.bkp`. Have used `-i.bkp` now

File: iofog-agent-daemon/src/main/java/org/eclipse/iofog/tracking/TrackingInfoUtils.java (hunk 1 above)
Comment (NehaNaithani) [reply iofog-agent-daemon/src/main/java/org/eclipse/iofog/tracking/TrackingInfoUtils.java]: We are using `secure_mode` for checking certs. If it’s on then we are making https calls. On the other hand we are using `dev_mode` for sending exceptions to sentry or not.

**Pull request merged**